.github/workflows/build.yml

```diff
@@ -30,22 +30,10 @@ jobs:
           toolchain: ${{ matrix.rust }}
       - run: rustup override set ${{ steps.toolchain.outputs.name }}
       - name: Build all features
-        uses: actions-rs/cargo@v1.0.1
-        with:
-          command: build
-          args: --all-targets --all-features
+        run: cargo build --all-targets --all-features
```

Comment:
I'm wondering whether it is useful to actually build the code here, or whether cargo check would suffice.

Reply:
I'm not entirely sure if `check` covers all the side-effects that `build` may have?: binary generation, post-processing steps on that binary, etc...